fdbclient/ServerKnobs.h

```diff
@@ -286,6 +286,7 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	int ROCKSDB_READ_QUEUE_HARD_MAX;
 	int ROCKSDB_FETCH_QUEUE_SOFT_MAX;
 	int ROCKSDB_FETCH_QUEUE_HARD_MAX;
+	double ROCKSDB_HISTOGRAMS_SAMPLE_RATE;
```

Comment:
can we add a comment: 
Measuring in hot path can cause performance overhead. 
Set to -1 to disable histogram.

Reply:
Done